Components/List/list.html

```diff
@@ -0,0 +1,203 @@
+						style="
+							width: 778px;
+							height: 661px;
+							border: 0;
+							transform: scale(1);
+							overflow: hidden;
+						"
+						sandbox="allow-scripts allow-same-origin"
+					>
+					</iframe>
+				</div>
+			</div>
+			<div class="container">
+				<p class="container-title">Stacked list</p>
+				<div class="stacked-container">
+					<ul class="stacked-list">
+						<li class="list-items">
+							<div class="card-horizontal">
+								<div class="card-img">
+									<img
+										src="/assets/images/luca-bravo-8Nql6oVT_1A-unsplash.jpg"
+										alt="card-img"
+									/>
+								</div>
+								<div class="card-contents">
+									<div class="card-title">
+										Card Title
+										<p class="sub-text">Sub-text</p>
+									</div>
+
+									<div class="card-description">Lorem Ipsem</div>
+								</div>
+							</div>
+						</li>
+						<li class="list-items">
+							<div class="card-horizontal">
+								<div class="card-img">
+									<img
+										src="/assets/images/luca-bravo-8Nql6oVT_1A-unsplash.jpg"
+										alt="card-img"
+									/>
+								</div>
+								<div class="card-contents">
+									<div class="card-title">
+										Card Title
+										<p class="sub-text">Sub-text</p>
+									</div>
+
+									<div class="card-description">Lorem Ipsem</div>
+								</div>
+							</div>
+						</li>
+					</ul>
+				</div>
+				<div class="code-snippet">
+					<iframe
+						src="https://carbon.now.sh/embed?bg=rgba%28245%2C245%2C245%2C0%29&t=one-light&wt=none&l=htmlmixed&width=809.5000009536743&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=JetBrains+Mono&fs=15.5px&lh=144%25&si=false&es=2x&wm=false&code=%2509%2509%253Cul%2520class%253D%2522stacked-list%2522%253E%250A%2509%2509%2509%253Cli%2520class%253D%2522list-items%2522%253E%250A%2509%2509%2509%2509%253Cdiv%2520class%253D%2522card-horizontal%2522%253E%250A%2509%2509%2509%2509%2509%253Cdiv%2520class%253D%2522card-img%2522%253E%250A%2509%2509%2509%2509%2509%2509%253Cimg%2520src%253D%2522%252Fassets%252Fimages%252Fluca-bravo-8Nql6oVT_1A-unsplash.jpg%2522%2520alt%253D%2522card-img%2522%252F%253E%250A%2509%2509%2509%2509%2509%253C%252Fdiv%253E%250A%2509%2509%2509%2509%2509%253Cdiv%2520class%253D%2522card-contents%2522%253E%250A%2509%2509%2509%2509%2509%2509%253Cdiv%2520class%253D%2522card-title%2522%253E%250A%2509%2509%2509%2509%2509%2509%2509Card%2520Title%250A%2509%2509%2509%2509%2509%2509%2509%253Cp%2520class%253D%2522sub-text%2522%253ESub-text%253C%252Fp%253E%250A%2509%2509%2509%2509%2509%2509%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2509%253Cdiv%2520class%253D%2522card-description%2522%253ELorem%2520Ipsem%253C%252Fdiv%253E%250A%2509%2509%2509%2509%2509%253C%252Fdiv%253E%250A%2509%2509%2509%2509%253C%252Fdiv%253E%250A%2509%2509%2509%253C%252Fli%253E%250A%2509%2509%253C%252Ful%253E"
```

Comment:
Could the carbon code snippet be modified in such a way that the list feature is show more prominently? as currently it just shows one card in a ul , the point of this list feature is to show multiple.